44702 Thousand separator in form inputs

dai-eastgate opened the pull request:
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/721
changed log:
Add thousand separator in form inputs

andernath commented on js/onoffice-apply-thousand-separator.js:
```diff
@@ -0,0 +1,58 @@
+
+    const getSeparators = (separatorFormat) => ({
+        thousandSeparator: separatorFormat === 'comma-separator' ? ',' : '.',
+        decimalSeparator: separatorFormat === 'comma-separator' ? '.' : ','
```
In https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/721#issuecomment-1966093746 you mention that you did not implement decimal separator. 
Does onoffice-apply-thousand-separator.js need some refactoring ?

andernath reviewed (commented):
@dai-eastgate looks good. I just have one question, could you take a look?

dai-eastgate commented:
@fredericalpers  @andernath I removed the decimal separator implementation
=> Currently the thousands separator feature will be automatic, users cannot enter "," or "." 
Ex:
With dot: 1000 -> 1.000; 123456 -> 123.456
With comma: 1000 -> 1,000; 123456 -> 123,456

Note: I have added a "Please choose" option for old users who are not using this feature

You can watch the demo video below:

https://github.com/user-attachments/assets/39a7ce94-eca8-4289-8d1a-cef8d1c157d4

miauzjpg reviewed (approved):
great work!
